WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -3,9 +3,12 @@ import Foundation
 
 extension NSString
 {
-    /**
-     *  @details Splits the lines contained in the current string, and returns its unique values in a NSSet instance.
-     */
+    /// Returns the string's hostname. If there's any
+    public func hostname() -> String {
+        return NSURLComponents(string: self as String)?.host ?? String()
+    }
```

Comment:
I'll update the `url.absoluteString.hostname()` calls. However, there are several other spots in the main app that don't deal with `NSURL`, so let's keep this, for the time being.

Thanks!